lib/utils/uid.js

```diff
@@ -4,8 +4,7 @@ let counter = 0
  * @link https://github.com/lodash/lodash/blob/4.17.4/lodash.js#L16143
  * @param {string} prefix
  */
-export default function uid(prefix = "") {
+export default function uid(prefix = "bv_") {
     const id = ++counter
```

Comment:
I've heard you talk about generating element id's before—what's the danger there?